### ISSUE TITLE
--module option doesn't work correctly with --start and --end options

### ISSUE THREAD

**Issue by akiym** [itchyny/rexdep#5]
Running [the example in README.md](https://github.com/itchyny/rexdep#extraction-range---start---end) now returns a different result.

v0.0.5 (maybe expected):
```
% rexdep_darwin_amd64/rexdep --pattern '"(\S+)"' --module '^package +(\S+)' --start '^import +\($' --end '^\)$' sample.go
main fmt
main github.com/urfave/cli
main os
```

v0.0.6 and latest v0.2.0:
```
% rexdep_darwin_amd64-1/rexdep --pattern '"(\S+)"' --module '^package +(\S+)' --start '^import +\($' --end '^\)$' sample.go
test.go fmt
test.go github.com/urfave/cli
test.go os
```

**Comment by itchyny** [comment]
I changed the behavior at 6dad67e0d3a3e8f2bc0cdcf722063d0dc339b147 for some reason, but I've forgot why. At lease I can see the inconsistency between readme and implementation so I fixed this now.

**Comment by itchyny** [comment]
Released as [v0.3.0](https://github.com/itchyny/rexdep/releases/tag/v0.3.0).

**Comment by akiym** [comment]
Thank you!
